### PR TITLE
Fix bug 1324936. Add link to resource in the translation tab.

### PR DESCRIPTION
**Pull request by jotes**
TODO:
* [x] - don't reload the whole page when the user clicks on a link inside the editor.

### CODE REVIEW

File: pontoon/base/static/js/translate.js [hunk 1]
```diff
@@ -3253,11 +3279,13 @@ var Pontoon = (function (my) {
         info: self.getProjectData('info'),
         width: self.getProjectWidth(),
         links: self.getProjectData('links') === 'True' ? true : false,
-        langpack_url: self.getProjectData('langpack_url')
+        langpack_url: self.getProjectData('langpack_url'),
+        parts: self.getProjectData('parts'),
```
Comment (mathjazz) [inline pontoon/base/static/js/translate.js]: We don't seem to use this property?

**Comment by jotes** [comment]
Yeah that's on my todo list.

1 maj 2017 11:10 AM "Matjaž Horvat" <notifications@github.com> napisał(a):

*@mathjazz* commented on this pull request.

Good job! fixed a few nits directly.

I have one suggestion only: for performance reasons, we should make the
AJAX request when the resource link is clicked. We can reuse code from the
#go button: https://github.com/jotes/pontoon/blob/
dee0c8fbf7844f725bb8938d0b435810d93a4c83/pontoon/base/
static/js/translate.js#L2704.
------------------------------

In pontoon/base/static/js/translate.js
<https://github.com/mozilla/pontoon/pull/592#discussion_r114110915>:

> @@ -3253,11 +3279,13 @@ var Pontoon = (function (my) {
         info: self.getProjectData('info'),
         width: self.getProjectWidth(),
         links: self.getProjectData('links') === 'True' ? true : false,
-        langpack_url: self.getProjectData('langpack_url')
+        langpack_url: self.getProjectData('langpack_url'),
+        parts: self.getProjectData('parts'),

We don't seem to use this property?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/mozilla/pontoon/pull/592#pullrequestreview-35552123>,
or mute the thread

**Comment by jotes** [comment]
@mathjazz  could you look again?u

**Comment by mathjazz** [comment]
r+ :)

Great job, @jotes!

**Pull request merged**